src/Lecture2.hs

```diff
@@ -193,8 +313,13 @@ verify that.
 >>> merge [1, 2, 4] [3, 7]
 [1,2,3,4,7]
 -}
+{- ok quite easy -}
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge [] bs = bs
+merge as [] = as
+merge (a : as) (b : bs)
+   | a < b     = a : merge as (b:bs)
+   | otherwise = b : merge (a:as) (bs)
```

Comment:
Absolutely correct solution! 💎 
You can make it more performant by checking if both heads of the given lists are equal. In that case, you're able to reduce the size of each list by 1 on each recursive call instead of reducing the size of only a single list ✂️